forum/communications/views.py

```diff
@@ -1,3 +1,41 @@
 from django.shortcuts import render
+from rest_framework import generics
+from rest_framework.permissions import IsAuthenticated
+from .models import Message
+from .serializers import ChatRoomSerializer, MessageSerializer
+from django.contrib.auth import get_user_model
 
-# Create your views here.
+User = get_user_model()
+
+
+def index(request):
+    return render(request, "communications/index.html")
+
+
+def room(request, room_name):
+    return render(request, "communications/room.html", {"room_name": room_name})
```

Comment:
Consider validation of `room_name` and handle cases where the room does not exist